easybuild/easyconfigs/s/SIP/SIP-4.19.25-foss-2022a-Python-3.10.4.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Bart Verleye
+# Center for eResearch, Auckland
+easyblock = 'ConfigureMakePythonPackage'
+
+name = 'SIP'
+version = '4.19.25'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
In the latest toolchains we dropped the Python version in `versionsuffix` (as long as you stick to the default Python version for that toolchain, which is the case here). So, this line can be removed, and the corresponding part should also be removed from the filename itself.

```suggestion
```